sycl/include/CL/sycl/ONEAPI/atomic.hpp

```diff
@@ -1,4 +1,4 @@
-//==---------------- atomic.hpp - SYCL_INTEL_extended_atomics --------------==//
+//==--------------- atomic.hpp - SYCL_ONEAPI_extended_atomics --------------==//
```

Comment:
If we're changing the name of the extensions as well as the namespace they're in, I think we should really update the extension documentation.

Reply:
Ok sure - but can that be a different PR?

Reply:
Fine by me, but I wanted to flag it somewhere.